nornir/plugins/inventory/netbox.py

```diff
@@ -135,6 +137,8 @@ def __init__(
         ssl_verify: Union[bool, str] = True,
         flatten_custom_fields: bool = False,
         filter_parameters: Optional[Dict[str, Any]] = None,
+        nb_private_key: Optional[str] = None,
+        nb_cred_role_slug: Optional[str] = "login-creds",
```

Comment:
I think we should use the name of a cred role, instead of the slug. The whole plugin relies on using the name of Netbox attributes instead of a the slug.

Reply:
Done.